lib/rubocop/cop/gusto/discouraged_gem.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Gusto
+      # Flags installation of discouraged gems (e.g., timecop) in Gemfiles and gemspecs.
+      #
+      # Configuration:
+      #   Gems:
+      #     timecop: "Use Rails' time helpers (e.g., freeze_time, travel_to) instead of Timecop."
+      #
+      # This cop is intended to be enabled in Rails projects via config/rails.yml.
+      class DiscouragedGem < Base
+        MSG = "Avoid using the '%{gem}' gem. %{advice}"
+
+        RESTRICT_ON_SEND = %i(gem add_dependency add_development_dependency).freeze
+
+        def on_send(node)
+          check_gem_usage(node)
+        end
+
+        private
+
+        def check_gem_usage(node)
+          gem_name = extract_gem_name(node)
```

Comment:
having written it out, it's not that much better, but something like this:
```suggestion
          return unless node.first_argument
          return unless node.first_argument.type?(:str, :sym)
          return unless discouraged_gems.include?(node.first_argument.value.to_s)
```